components/table/ReportTable.tsx

```diff
@@ -58,6 +59,13 @@ interface ReportTableProps {
 const entriesQuantitySum = (entries: TimesheetEntry[]) =>
     entries.reduce((total, currentItem) => total + currentItem.quantity, 0)
 
+const totalRate = (entries: TimesheetEntry[]) =>
```

Comment:
I think `totalIncome` would be more apt. I think rate should be measured in `€/h` while income is in `€`.